install.sh

```diff
@@ -304,8 +304,7 @@ install_deb () {
   echo "Updating package indexes..."
   ${SUDO} apt-get update
   echo "Installing ${installer_name}..."
-  ${SUDO} apt-get install ${yesapt} gdebi-core
-  ${SUDO} gdebi ${yes} "${installer_name}"
+  ${SUDO} apt-get install ${yes} "${installer_name}"
```

Comment:
Looks like this is failing in tests from ending up as `apt-get install --n`. First, `--n` looks like a bug but it seems unused, could be removed anyway? And then I believe we need `$yesapt` instead here.